core/src/scene/filters.h

```diff
@@ -26,15 +26,23 @@ enum class FilterType : int {
     undefined
 };
 
+enum class FilterGlobal : int {
```

Comment:
+1, this is problem necessary for eraser map styling after some scene and tile refactor.

Sent from my handsful device.



> On Oct 28, 2015, at 12:50, Varun <notifications@github.com> wrote:
> 
> In core/src/scene/filters.h:
> 
> > @@ -26,15 +26,23 @@ enum class FilterType : int {
> >      undefined
> >  };
> >  
> > +enum class FilterGlobal : int {
> We can have this as a char instead of an int.
> 
> ―
> Reply to this email directly or view it on GitHub.
>